python/get_allrevisions.py

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/python3
+
+"""
+    get_allpages.py
```

Comment:
File name and description here are different than what you've added in `modules.json` file. It looks like didn't autogenarate this file..